### PR TITLE
Add test for AddCommand

### PR DESCRIPTION
**Pull request by aidoxe-123**
Improve test coverage

### CODE REVIEW

File: src/test/java/jimmy/mcgymmy/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -42,6 +45,25 @@ public void execute_foodAcceptedByModel_addSuccessful() throws Exception {
...
+            new CommandParserTestUtil.OptionalParameterStub<>("p", validFood.getProtein()),
+            new CommandParserTestUtil.OptionalParameterStub<>("f", validFood.getFat()),
+            new CommandParserTestUtil.OptionalParameterStub<>("c", validFood.getCarbs()),
+            new CommandParserTestUtil.OptionalParameterStub<>("t", new Tag("hello"))
+        );
+
+        CommandResult commandResult = command.execute(modelStub);
+
+        assertEquals(String.format(AddCommand.MESSAGE_SUCCESS, validFood), commandResult.getFeedbackToUser());
+        assertEquals(Arrays.asList(validFood), modelStub.foodAdded);
```
Comment (Jh123x) [inline src/test/java/jimmy/mcgymmy/logic/commands/AddCommandTest.java]: Might be able to use CommandTestUtil here to reduce code duplication.
Can compare the tags after the CommandTestUtil.assertCommandSuccess

**Review by Jh123x — CHANGES_REQUESTED** [review]

File: src/test/java/jimmy/mcgymmy/logic/commands/AddCommandTest.java (hunk 1 above)
Comment (aidoxe-123) [reply src/test/java/jimmy/mcgymmy/logic/commands/AddCommandTest.java]: Fixed

**Review by Jh123x — APPROVED** [review]
LGTM ! Just need to resolve the conflict

**Review by Jh123x — APPROVED** [review]
LGTM!

**Pull request merged**